docs/080-breaking-changes.rst

```diff
@@ -86,12 +89,16 @@ New Restrictions
   This will make it safe to assume that the underlying type in the ABI is always ``uint8``.
 
 * Declarations with the name ``this``, ``super`` and ``_`` are disallowed, with the exception of
-  public functions and events.
+  public functions and events to make it possible to interface with languages other than Solidity.
 
 * Remove support for the ``\b``, ``\f``, and ``\v`` escape sequences in code.
   They can still be inserted via hexadecimal escapes, e.g. ``\x08``, ``\x0c``, and ``\x0b``, respectively.
 
 Interface Changes
```

Comment:
Mention the error reporter?

Reply:
Oh, that is not even in the changelog, is it?